README.md

```diff
@@ -1,2 +1,6 @@
 # camt053
 This repository contains generated Java classes to read and write the camt053 format.
+
+The classes were generated from camt.053.001.02 XSD file downloaded from https://www.iso20022.org/catalogue-messages/iso-20022-messages-archive?search=camt#Bank2CustomerCashManagementv2
+
+Tool used for generation of JAXB-Classes is XJC 3.0.0 downloaded from https://github.com/eclipse-ee4j/jaxb-ri/releases
```

Comment:
Please change to
The tool used for generation of JAXB-Classes is XJC 3.0.0 downloaded from https://github.com/eclipse-ee4j/jaxb-ri/releases.